anybox/recipe/odoo/runtime/session.py

```diff
@@ -207,7 +208,11 @@ def init_environments(self):
         self._environments_gen_context = gen_factory().gen
         self._environments_gen_context.next()
         if hasattr(openerp, 'api'):
```

Comment:
by the way, since a.r.odoo is for Odoo >=8, this is always true, now
I suspect some people are using it for OpenERP 7, though